src/Frontend/styles/custom/navigation.less

```diff
@@ -18,4 +18,9 @@ body {
   #content {
     margin-top: 70px;
   }
+  ul.dropdown-menu a:hover {
+    background-color: transparent;
+    border: none;
+    outline: none;
+  }
```

Comment:
you should add a custom class to the ``load-languages`` item since this would avoid any hover even on the actual language list where the hover should exist